### PR TITLE
CI: add test ci

### PR DESCRIPTION
**Pull request by gxthrj**

### CODE REVIEW

File: .github/workflows/test-ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup go
+      uses: actions/setup-go@v1
+      with:
+        go-version: '1.13'
+    - name: run test
+      working-directory: ./
+      run: |
+        go test ./...
```
Comment (tokers) [inline .github/workflows/test-ci.yml:23]: Please check the editor, always add the linebreak.

File: pkg/ingress/apisix/upstream_test.go [hunk 2]
```diff
@@ -94,6 +97,7 @@ kind: ApisixUpstream
 apiVersion: apisix.apache.org/v1
 metadata:
   name: httpserver
+  namespace: cloud
```
Comment (tokers) [inline pkg/ingress/apisix/upstream_test.go:100]: Still use cloud namespace? we should use ingress-apisix.

Comment (gxthrj) [reply pkg/ingress/apisix/upstream_test.go:100]: This is a test-case

File: .github/workflows/test-ci.yml (hunk 1 above)
Comment (gxthrj) [reply .github/workflows/test-ci.yml]: Done

File: pkg/ingress/apisix/upstream_test.go (hunk 2 above)
Comment (tokers) [reply pkg/ingress/apisix/upstream_test.go:100]: OK

File: .github/workflows/test-ci.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: test-ci
```
Comment (tokers) [inline .github/workflows/test-ci.yml:1]: the name test-ci is too generic, actually it's just unit test, we will have e2e tests in the future, so `unit-test.yaml` is better.

Comment (gxthrj) [reply .github/workflows/test-ci.yml]: ok, use unit-test-ci.yml

**Comment by tokers** [comment]
LGTM

**Review by tokers — APPROVED** [review]

**Comment by membphis** [comment]
nice

**Pull request merged**